tests/python/pytest_opinfos.py

```diff
@@ -1103,6 +1105,24 @@ def torch_reshape_sym_fn(input_tensor, output_shaped_tensor):
 )
 tensor_creation_ops.append(uniform_opinfo)
 
+matmul_ops = []
+
+
+def matmul_wrapper(fd, a, b):
+    t1 = fd.ops.matmul(a, b)
+    t2 = fd.ops.cast(t1, dtype=DataType.Half)
```

Comment:
Should this also include bfloat16?

Reply:
Thanks, I added this, but the type deduction is clumsy. I'd need further PR(s) to unify the tests.